### DISCUSSION TITLE
[v5 - release] - How to use count on models?

### DISCUSSION THREAD

**Discussion by sn0rcha** [adonisjs/core#2549] [Help] [answered]
In the preview release, we could do the following:

```
const count = await SomeModel.query().count('id')
console.log(count[0].count)
```
And it would show the value. However, in release, I'm getting ```undefined``` with the same code.

Any ideas on how to access the count now?

UPDATE: I noticed if I do ```console.log(count[0].$extras)``` it's in there - is that correct?

Cheers,
Ben

**Comment by thetutlage** [comment, marked as answer]
Use the `pojo` method. Documented here https://docs.adonisjs.com/reference/orm/query-builder#pojo

```ts
const count = await SomeModel.query().pojo<{ total: number }>().count('id as count')
console.log(count[0].count)
```

**Reply by sn0rcha** [reply]
Oh awesome, thanks!

I'm also getting this warning

```
"Model.preload()" is deprecated. Use "Model.load()" instead: DeprecationWarning
```

But when I use .load() it's throwing a TS error - is that correct?
